rasa/core/policies/policy.py

```diff
@@ -293,6 +293,78 @@ def _default_predictions(domain: Domain) -> List[float]:
         return [0.0] * domain.num_actions
 
 
+class PolicyPrediction:
+    """Stores information about the prediction of a `Policy`."""
+
+    def __init__(
+        self,
+        probabilities: List[float],
+        policy_priority: int,
+        events: Optional[List[Event]] = None,
+        optional_events: Optional[List[Event]] = None,
+        is_end_to_end_prediction: bool = False,
```

Comment:
should we also include the policy name here?

Reply:
I think so, since we any how return a structure, why not add additional field, then return of several methods can be simplified